specifications/api/swagger.yaml

```
@@ -5200,11 +5200,33 @@ x-errPoolAlreadyJoined: &errPoolAlreadyJoined
   properties:
     message:
       type: string
-      description: May occur when a given poolId matches the current delegation preferences of the wallet's account.
+      description: May occur when a given poolId matches the current delegation preferences of the wallet's account (not in Conway onwards provided different vote was also casted).
     code:
       type: string
       enum: ['pool_already_joined']
 
+x-errPoolAlreadyJoinedSameVote: &errPoolAlreadyJoinedSameVote
+  <<: *responsesErr
+  title: pool_already_joined_same_vote
+  properties:
+    message:
+      type: string
+      description: May occur in Conway onwards when a given poolId matches the current delegation preferences of the wallet's account and also the same vote was casted.
+    code:
+      type: string
+      enum: ['pool_already_joined_same_vote']
+
+x-errSameVote: &errSameVote
+  <<: *responsesErr
+  title: same_vote
+  properties:
+    message:
+      type: string
+      description: May occur in Conway onwards when the same vote was casted.
```

```suggestion
      description: May occur in Conway onwards when the same vote was cast.
```

The past participle is "cast".